### ISSUE TITLE
can not use for pytorch version >=0.30，the var.grad_fn's name and its params are different

### ISSUE THREAD

**Issue by wangbofei11** [marvis/pytorch-caffe-darknet-convert#39]
In the newest version of pytorch, many grad_fn's name changed including conv、batchnorm、pooling ……for example,ConvNDBackward changed to Thnn2DBackward, 
I changed these names with the newest version, but another problem appear, the params of grad_fn also changed. for example, func.padding、func.stride、func.dilation、func.running_mean…… are not exist.
Until now, I have not found a solution, could anyone get help? tks

**Comment by kyo055** [comment]
@wangbofei11  Hi, I also find the same problem.  Have you found the problem solution? THX
